scripts/backfill_cdc.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+# Backfills all the CDC tables
+set -e
+
+declare -a STORAGES=("groupedmessages"  "groupassignees")
+
+log_message() {
+    GREEN='\033[0;32m'
+    NC='\033[0m'
+
+    echo -e "$GREEN"
+    echo "$1"
+    echo -e "$NC"
+}
+
+mkdir -p /tmp/cdc-snapshots/
+
+GREEN='\033[0;32m'
+NC='\033[0m'
```

Comment:
It seems these two are redeclared within `log_message`. Can you declare them at the top of this file only once?